vms/platformvm/service_test.go

```diff
@@ -171,6 +171,7 @@ func TestGetTxStatus(t *testing.T) {
 	defaultAddress(t, service)
 	service.vm.ctx.Lock.Lock()
```

Comment:
nit: just for clarify, I'd `moveservice.vm.ctx.Lock.Lock()` needed to carry out Apply over peerShareMemory, to line 207

Reply:
I tried to only touch locks that I needed to touch for this PR